src/ovp_pipeline/prompt_registry.py

```
@@ -0,0 +1,280 @@
+        return sorted(p.name for p in self._prompts_dir.iterdir() if p.is_dir())
+
+    # ------------------------------------------------------------------
+    # Internal helpers
+    # ------------------------------------------------------------------
+
+    @staticmethod
+    def _parse(path: Path, expected_name: str, expected_version: str, text: str) -> Prompt:
+        if not text.startswith("---"):
+            raise PromptRegistryError(
+                f"{path}: missing YAML frontmatter (file does not start with '---')"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)